README.rst

```diff
@@ -39,16 +39,22 @@ Features
 
     <img src="https://raw.githubusercontent.com/XanaduAI/strawberryfields/master/doc/_static/StateLearning.gif" width="300px"  align="right">
 
-* An open-source software architecture for **photonic quantum computing**
+* Execute photonic quantum algorithms directly on `Xanadu's next-generation quantum hardware.
+  <https://strawberryfields.ai/photonics/hardware/index.html>`__
 
-* Execute algorithms on  **photonic quantum computing chips** via the Xanadu Quantum Cloud
+* `High-level functions <https://strawberryfields.ai/photonics/applications.html>`__ for **solving
+  practical problems** including **graph and network optimization**, **machine learning** and
+  **chemistry**
 
-* Includes high-level functions for **solving practical problems** including **graph and network
-  optimization**, **machine learning** and **chemistry**
+* Includes a suite of world-class simulators—based on cutting edge algorithms—to compile and
+  simulate photonic algorithms.
 
-* Includes **quantum simulators** implemented using **NumPy** and **TensorFlow**—with built-in quantum compilation
+* `Train and optimize
+  <https://strawberryfields.ai/photonics/demonstrations.html#optimization-and-machine-learning>`__
```

Comment:
Title cut off when linked page is loaded :(

Reply:
:(